### ISSUE TITLE
Player Specific Placeholders Not Working on Reward Executions.

### ISSUE THREAD

**Issue by HeavenCrafter** [PyvesB/AdvancedAchievements#729]
In the plugin DiscordSRV, there is a command called /discordsrv bcast (message) which will make the bot send a message to the appropriate channel.

I wanted to use that command on custom achievements rewards command execution.

Although the command itself worked perfectly fine, and the reward system did trigger, the player placeholder itself, did not work.

This was my configuration -
```
Commands:
  yourAch1:
    Goal: Unlock the final community center bundle.
    Message: You've done it all.
    Name: your_ach1
    DisplayName: Master Achiever!
    Reward:
        Command:
          Execute: "discordsrv bcast #707269837646200845 Player %player_name% on **Survival** has gained the ***Master Achiever Title***!"
```

%player_name% or any other PAPI player specific placeholder will come up as empty. However, anything that ISN'T player specific (such as server ram or server tps placeholders) works completely fine.

**Comment by PyvesB** [comment]
Hello @HeavenCrafter , I suggest checking out the [Wiki](https://github.com/PyvesB/AdvancedAchievements/wiki/Rewards#custom-commands). :wink:

**Comment by HeavenCrafter** [comment]
I cannot tell if I'm annoyed with myself, or annoyed with how tired I am.

I did %player_name% the first time and it didn't work so I was like okay, maybe it was something from DiscordSRV. But its not.

And then I started looking at the wiki, I saw the same exact placeholder, and I did it, and it didn't work.

BUT THE MOMENT I RESTART THE SERVER, IT STARTS WORKING.

I am going to yeet myself across the god damn lake.

Thank you for responding though, guess it solved itself. That was a waste of a days time.
